src/tv2-common/getShowStyleVariantId.ts

```diff
@@ -0,0 +1,20 @@
+import {
+	IBlueprintShowStyleVariant,
+	IngestRundown,
+	IStudioUserContext
+} from '../../../tv-automation-server-core/packages/blueprints-integration'
+
+const DEFAULT_VARIANT_NAME = 'default'
```

Comment:
Is this documented somewhere, that the _Default_ showstyle variant name becomes special?
We also have [some migrations](https://github.com/tv2/sofie-blueprints-inews/blob/develop/src/tv2_afvd_showstyle/migrations/variants-defaults.ts#L18-L24) creating a variant called _Main_. Shouldn't we get rid of that one?

Reply:
No, I dont think it is documented anywhere. It was decided by Jacob and the producer Rasmus while testing in gallery D.

I think we can remove the migration creating the 'Main' variant since I don't see us ever using the 'Main' variant.